src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -192,10 +239,24 @@ private void saveElectionConfig(File saveFile) {
 
     config.candidates = new ArrayList<>(tableViewCandidates.getItems());
 
+    rules.tiebreakMode =
+        choiceTiebreakMode.getValue() != null
+            ? choiceTiebreakMode.getValue().toString()
+            : Tabulator.TieBreakMode.MODE_UNKNOWN.toString();
     rules.overvoteRule =
         choiceOvervoteRule.getValue() != null
             ? choiceOvervoteRule.getValue().toString()
             : Tabulator.OvervoteRule.RULE_UNKNOWN.toString();
+    rules.multiSeatTransferRule =
+        choiceMultiSeatTransferRule.getValue() != null
+            ? choiceMultiSeatTransferRule.getValue().toString()
+            : Tabulator.MultiSeatTransferRule.TRANSFER_RULE_UNKNOWN.toString();
+    rules.maxRankingsAllowed = Integer.parseInt(textFieldMaxRankingsAllowed.getText());
```

Comment:
Will these trigger an uncaught exception if the text isn't a valid integer? Do your listeners make that truly impossible, or just unlikely?

Reply:
Yes, it's impossible. Anything that doesn't match regex "\\d*" in these text fields is immediately reverted, as if it was never typed at all. Maybe not the best UX (ideally we'd have a pop-up note that says "integers only"), but it'll do the trick in the short-term.